eras/alonzo/test-suite/cardano-ledger-alonzo-test.cabal

```
@@ -45,15 +45,15 @@ library
     build-depends:
         base >=4.14 && <4.19,
         bytestring,
-        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.5.1,
+        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.6,
         cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.0,
-        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.5 && <1.10,
+        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.10 && <1.11,
```

Lower bound change is not needed, it will work fine with the older version.
You only need to update the lower bounds in the package if this package now relies on some changes in the dependency, otherwise you are over-restricting the bounds

```suggestion
        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.5 && <1.11,
```